deployment/requirements/dev.txt

```diff
@@ -120,6 +120,68 @@ isort==5.10.1 \
 jedi==0.18.1 \
     --hash=sha256:637c9635fcf47945ceb91cd7f320234a7be540ded6f3e99a50cb6febdfd1ba8d \
     --hash=sha256:74137626a64a99c8eb6ae5832d99b3bdd7d29a3850fe2aa80a4126b2a7d949ab
+lxml==4.8.0 \
```

Comment:
There is no need to duplicate this dependency between `dev.txt` and `prod.txt` (remove lxml from `dev.in` and run `make compile-deps`)